hedera-node/hedera-mono-service/src/main/java/com/hedera/services/store/contracts/AbstractLedgerWorldUpdater.java

```diff
@@ -338,6 +359,11 @@ protected UpdateTrackingLedgerAccount<A> track(final UpdateTrackingLedgerAccount
         return account;
     }
 
+    public void trackLazilyCreatedAccount(final Address address) {
```

Comment:
I don't think this will be necessary if we update `onAccountPropertyChange()` [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/services/store/contracts/AbstractLedgerWorldUpdater.java#L295) to account for account creations.